docker-compose-pool.yaml

```diff
@@ -54,6 +54,9 @@ services:
         - https_proxy
         - no_proxy
         - ENCLAVE_TYPE=wpe
+        # Configure list of workload that need to be supported
+        # by this WPE by separating each workload by delimiter semicolon.
+        - WORKLOADS=echo-result;heart-disease-eval
```

Comment:
Pass "all" by default here. Any reason why we use semi-colon ( ; ) specifically. I ask this as I made use of comma ( , ) in the config files. Also, a reference here or in the config file would help saying that this config needs to be same at either places.

Reply:
i added the two workloads because they are majorly used and it demonstrates how to pass the params. 
semicolon is passed between the workloads because cmake treats it as list splits the workloads based on semicolon.

Reply:
Added all the workloads as build argument so that WPE supports all workloads by default.